packages/utils/src/device.ts

```diff
@@ -0,0 +1,9 @@
+export const isDeviceBitcoinOnly = (device: any) => {
```

Comment:
@trezo/utils is really not a good place for this, we need to find/create better place - this is not the first function we need to share between Suite and Connect. Usually we put those in Connect, I guess. But where?

side note:
It's better to prefer `unknown` over `any` type. `unknown` is the type-safe counterpart of `any`... And if the type is unspecified, we should handle all possible types, so handle error if `device` is `undefined`...

Reply:
good catch. it should not be `any`. I think it should be known type, something like `TrezorDevice`. I did not fix that since I was quite sure that I will be touching it anyway cause I did not like placing it into @trezor/utils either but I did not know where it should go. 

I am against exporting utilities from @trezor/connect. Does not feel right either.

Maybe we could get rid of  this utility completely and add a field to device object in connect?. Something like device.isBtcOnly?

Reply:
device.isBitcoinOnly sounds good to me!

Reply:
added 3b2bb03a49deafc2906a0342717f3e22e20482f7. imho better to add firmwareType = 'regular' | 'bitcoin-only' as it is more extensible

Reply:
Is there no need for db migration?

Reply:
huh, very good point! There is. I am also interested if this was caught by migration test or not! cc @ondracja

Reply:
it is passing :( https://gitlab.com/satoshilabs/trezor/trezor-suite/-/jobs/2953167935/artifacts/file/packages/integration-tests/projects/suite-web/videos/suite/database-migration.test.ts.mp4 
we don't have bitcoin-only emulator at the moment, buuut it still could be caught somewhere maybe? :thinking: